admin_guide/router.adoc

```diff
@@ -10,45 +10,47 @@
 toc::[]
 
 == Overview
-Once you have a running link:../architecture/core_objects/routing.html[Router] you are ready
-to monitor it.  Depending on the underlying router implementation this may be accomplished
-multiple ways.  This document will discuss the HAProxy template router and the different
-components that can be checked to ensure its health.
+Depending on the underlying implementation, you can monitor a running
+link:../architecture/core_objects/routing.html[router] in multiple ways. This
+topic discusses the HAProxy template router, and the components to check to
+ensure its health.
 
 == Viewing Statistics
-The HAProxy router exposes a web listener for the HAProxy statistics.  You can enter the public
-IP address of the router with the correctly configured port to view the statistics page.  You 
-will be prompted for the admin password.  This password and port were configured during the
-router installation.
+The HAProxy router exposes a web listener for the HAProxy statistics. Enter the
+router's public IP address and the correctly configured port (1936 by default)
+to view the statistics page. You will be prompted for the admin password. This
+password and port were configured during the router installation, but can be
+found by viewing the *_haproxy.conf_* file on the container.
 
 == Viewing Logs
-To view a router log, simply run the `osc log` command for the pod.  Keep in mind that since
-the router is running as a plugin process that manages the underlying implementation the log
-that you will see is the log for the plugin, not the actual HAProxy log.  
+To view a router log, run the `osc log` command on the pod. Note that, since the
```

Comment:
It's generally not a good idea to start something with "Note that", and if you really feel strongly that it should, then you should make it an admonition NOTE. I think you could:

I would change this: Note that, since the router is running as a plugin process...

To this: The router is running as a plug-in process that manages the underlying implementation, so the log is for the plug-in, not the actual HAProxy log.